charts/cni-metrics-helper/Chart.yaml

```diff
@@ -20,4 +20,4 @@ version: 0.1.8
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: v1.11.0
+appVersion: v1.11.1
```

Comment:
Even here need to increment the chart version.

Reply:
Updated here - https://github.com/aws/amazon-vpc-cni-k8s/pull/1995 

branch protection prevented my update in this PR.